### ISSUE TITLE
redis can not start success-

### ISSUE THREAD

**Issue by edidada** [kumahq/kuma-demo#81]
vagrant up
and See error
redis not start
and I vagrant ssh to redis vm,run script, I found script can not make directory in /home/vagrant

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Kuma: [e.g. v0.3.1]
 - Environment [e.g. Kubernetes, Universal]

**Additional context**

**Comment by roverliang** [comment]
I hava the same probem .

error msg :

```bash
 redis: Running: /var/folders/ql/9lb5vf4s6hz5xv7m7h12jbjh0000gn/T/vagrant-shell20200806-20264-1uh6sk0.sh
    redis: Error: Could not create a resource (Resource is not valid)
    redis: * networking.inbound[0].tags["kuma.io/service"]: tag has to exist
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by devadvocado** [comment]
the tag issue is now resolved thanks to pr #86 by @jakubdyszkiewicz. Tags had some breaking change in v0.7.0, apologizes for not updating this part the demo to fix it.

**Comment by devadvocado** [comment]
along with pr #88 that I just merged, should fix any remaining issues with running the demo on universal. once again, sorry for the inconvenience folks!

I'll close off this issue for now. If you encounter anything else, feel free to create a new issu
